contrib/lisp/org-roam-helm-bibtex.el

```diff
@@ -0,0 +1,40 @@
+;;; org-roam-helm-bibtex.el --- Org-roam helm-bibtex integration  -*- fill-column: 78; lexical-binding: t; -*-
+
+;; Authors: Leo Vivier <leo.vivier+dev@gmail.com>
+
+;;; Commentary:
+
+;; This script provides basic interaction between `org-roam' and `helm-bibtex'
+;; by enabling bibliographic notes stored in `org-roam-directory' to be
+;; accessed from `helm-bibtex' or `ivy-bibtex'.  If a note does not exist, the
+;; user is asked to create one.
+
+;;; Code:
+
+(require 'bibtex-completion)
+
+(with-eval-after-load 'helm
+  (require 'helm-bibtex))
+
+(with-eval-after-load 'ivy
+  (require 'ivy-bibtex))
+
+(defun org-roam-bibtex-completion-edit-notes (keys)
+  "Open the notes associated with the selected entries using `find-file'."
+  (dolist (key keys)
+    (let ((refs (org-roam--get-ref-path-completions)))
+      (if-let ((path (cdr (assoc key refs))))
+          (find-file path)
+        (when (y-or-n-p (format "No note was found for %s.  Would you like to create one?" key))
```

Comment:
Do you think this prompt is necessary? It's an additional step for users, and the capture process can always be cancelled.

Reply:
I don't think it's necessary.  Since `helm-bibtex` shows you in its main-window whether an entry has a note associated to it, I think it's fair to assume that the user /wants/ to create one if it doesn't exist.  I'll patch it out now.